tests/requirements.txt

```diff
@@ -0,0 +1 @@
+toml
```

Comment:
I think you should use `pipreqs` or similar to have detailed package requirement and not get jebaited when `toml` is updated without support for older versions.

Reply:
I'll add a stricter version requirement